### DISCUSSION TITLE
2.12.2

### DISCUSSION THREAD

**Discussion by nikosgram** [nikosgram/gringotts#74] [Announcements]
* Removes server balance and vault count from bstats reports
* Improve UUID support
* Clean up code

**Full Changelog**: https://github.com/nikosgram/gringotts/compare/2.12.1...2.12.2

<hr /><em>This discussion was created from the release <a href='https://github.com/nikosgram/gringotts/releases/tag/2.12.2'>2.12.2</a>.</em>

**Comment by spikehidden** [comment]
What's the difference between "Original Gringotts" and "Gringotts"?

**Reply by nikosgram** [reply]
Original gringotts is just the source :) Without dependencies
